hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx

```diff
@@ -77,10 +81,49 @@ export const useCompletenessStatsColumns = (params: any) => {
                 accessor: 'forms_filled_direct',
                 sortable: false,
                 Cell: settings => (
-                    <span>
-                        {settings.row.original.forms_filled_direct ?? '--'}/
-                        {settings.row.original.forms_to_fill_direct ?? '--'}
-                    </span>
+                    <Box>
+                        <div
+                            style={{
+                                // marginTop: '2px', // The margin will align the icon in FF but misalign it in Chrome
+                                display: 'inline-flex',
+                                alignItems: 'center',
+                                verticalAlign: 'middle',
+                            }}
+                        >
+                            {`${
+                                settings.row.original.forms_filled_direct ??
+                                '--'
+                            }/
+                            ${
+                                settings.row.original.forms_to_fill_direct ??
+                                '--'
+                            }`}
+                        </div>
+                        {settings.row.original
+                            .has_multiple_direct_submissions && (
+                            <Tooltip
+                                title={formatMessage(
+                                    MESSAGES.orgUnitHasMultipleSubmissions,
+                                )}
+                            >
+                                <div
+                                    style={{
+                                        display: 'inline-flex',
+                                        alignItems: 'center',
+                                        verticalAlign: 'middle',
+                                    }}
+                                >
+                                    <AddCircleOutlineIcon
```

Comment:
I don't understand the meaning of this icon, there is no action or `Tooltip` linked to it

Reply:
There is a tooltip:

![Screenshot 2023-02-15 at 10 43 19](https://user-images.githubusercontent.com/38907762/218991660-29ae859b-b7ca-40f3-9c3c-83fa9aa6a525.png)